### ISSUE TITLE
System freeze randomly

### ISSUE THREAD

**Issue by kepler-dslr** [elementary/stylesheet#598]
`Mar 27 18:58:46 kepler-X540LJ gala[1468]: message repeated 5 times: [ Utils.vala:158: Icon 'application-default-icon' not present in theme Win10X]
Mar 27 18:59:12 kepler-X540LJ gala.desktop[1468]: Window manager warning: Tried to ping a window with a bad serial! Not allowed.
Mar 27 18:59:20 kepler-X540LJ plank.desktop[1487]: [19818:1:0327/185920.071505:ERROR:child_process_sandbox_support_impl_linux.cc(79)] FontService unique font name matching request did not receive a response.`

That's the syslog from the moment it happens, I reinstalled and still the same thing

**Comment by cassidyjames** [comment]
It looks like you're using a third-party icon set, which is not supported on elementary OS. Is this hang happening when you use the default OS settings? Does it only happen with a certain app?

**Comment by kepler-dslr** [comment]
Yeah, in fact, yesterday I reinstalled the OS, until this freezes made their appearance everythng was so perfect and now this.

**Comment by kepler-dslr** [comment]
It ocurrs mostly when I write something on browser and music is running in another tab
